PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,11 @@
+IMPORTANT NOTE: Please do not remove the 3 headings below (Changes Proposed, Important Links, Additional Info). Just fill in what is needed. (You can remove this line)
+
+### Changes Proposed in this Pull Request:
+* 
+* 
+
+### Important Links (Pivotal, Additional Repositories, etc):
+
```

Comment:
Replace `Pivotal` with JIRA`. Also could use one default * (and remove the excess one from the previous heading)